FF1Lib/RomStampModel.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Schema;
+using System.IO;
+
+namespace FF1Lib
+{
+	public class RomStampModel
```

Comment:
I don't love the name RomStampModel, Why not something like BuildInfo or DebugDigest. 

The class isn't really specific to ROMs. Stamp is somewhat illustrative of the purpose. "Model" at the end isn't super helpful either.

Reply:
I think my favorite after mulling it about would be `RandomizeDigest` since it's essentially a digest of the result of the call to Randomize.

Then you could define it inline right above the Randomize method if you like.

It's fine in its own file too, but surely you can chuck all those ugly using statements!